src/pybind/mgr/cephadm/nfs.py

```diff
@@ -46,6 +46,14 @@ def create_keyring(self):
         })
         return keyring
 
+    def create_rados_config_obj(self):
+        # type: () -> None
+        obj = self.get_rados_config_obj()
+        with self.mgr.rados.open_ioctx(self.pool) as ioctx:
+            if self.namespace:
+                ioctx.set_namespace(self.namespace)
+            ioctx.write_full(obj, ''.encode('utf-8'))
```

Comment:
I think this should only create the object if it doesn't already exist.  Otherwise if we re-execute this for some reason we will clobber existing config state....

Reply:
will assume an existing config and leave untouched if `stat` returns a result